src/MaterialEditor.Core.Studio/Core.MaterialEditor.SceneController.cs

```diff
@@ -30,13 +32,13 @@ public class SceneController : SceneCustomFunctionController
         private readonly List<MaterialFloatProperty> MaterialFloatPropertyList = new List<MaterialFloatProperty>();
         private readonly List<MaterialKeywordProperty> MaterialKeywordPropertyList = new List<MaterialKeywordProperty>();
         private readonly List<MaterialColorProperty> MaterialColorPropertyList = new List<MaterialColorProperty>();
-        private readonly List<MaterialTextureProperty> MaterialTexturePropertyList = new List<MaterialTextureProperty>();
+        internal readonly List<MaterialTextureProperty> MaterialTexturePropertyList = new List<MaterialTextureProperty>();
```

Comment:
Not a fan of exposing internals specific to scenes, ideally there would be a cleaner way to do it.